patchwise/patchwise_config.yaml

```
@@ -0,0 +1,3 @@
+open_ai:
+  key_agreement: false
+  show_again: true
```

We don't need to keep track of the key_agreement itself in the config file, we just need a config for showing the disclaimer.

```suggestion
show_key_disclaimer: false
```